jenkins.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
It's to avoid duplicate in various Jenkins jobs. We can just invoke this script and pass any extra parameters (like Scala version).